packages/api/src/services/contacts.service.ts

```diff
@@ -15,19 +16,38 @@ export class ContactsService {
         return this.contactRepository.find({ where: { userId }, order: { contactName: "ASC" } });
     }
 
+    async findSpecific(userId: number, contactName: string) {
```

Comment:
Name and return type

```suggestion
    async find(userId: number, contactName: string): Promise<Contact> {
```